spec/client_spec.rb

```diff
@@ -1,3 +1,4 @@
+# coding: utf-8
```

Comment:
I don't think this needs to be here? Not that it's doing any harm...

Reply:
my editor is very insistent on adding it :/ I'll keep it if that's ok